src/Npgsql/NpgsqlCommand.cs

```diff
@@ -1745,13 +1741,13 @@ internal void TraceCommandEnrich(NpgsqlConnector connector)
             NpgsqlActivitySource.Enrich(CurrentActivity, connector);
             var tracingSettings = connector.DataSource.Configuration.TracingOptions;
```

Comment:
Any reason we're not using the parameter here? Or alternatively if we can pull everything from the datasource any reason we have parameters on this method?

Reply:
I could swear I added the comment on another line, see update